### PR TITLE
Sorting program lists

### PR DESCRIPTION
**Pull request by mlehtima**
Some other small fixes.

### CODE REVIEW

File: qml/pages/ProgramsPage.qml [hunk 1]
```diff
@@ -12,6 +12,7 @@ Page {
     property int offset: 0
     property int limit: 25
     property bool programsEnd: false
+    property string order: 'publication.starttime:desc'
```
Comment (joonne) [inline qml/pages/ProgramsPage.qml:15]: Could this be a property inside `yleApi`? Feels like a better place for it, or even all the others above could be moved there as "constants"? :thinking:

File: qml/pages/ProgramDelegate.qml [hunk 2]
```diff
@@ -7,6 +7,7 @@ ListItem {
     id: listItem
     contentWidth: ListView.view.width
     contentHeight: Theme.itemSizeLarge
+    property bool seriesList: false
```
Comment (joonne) [inline qml/pages/ProgramDelegate.qml]: rename to `isSeriesList`?

**Review by joonne — APPROVED** [review]
Good stuff reading the API docs properly and adding the order :+1:

File: qml/pages/ProgramsPage.qml (hunk 1 above)
Comment (mlehtima) [reply qml/pages/ProgramsPage.qml:15]: This is constant only for now, later I will add settings for selecting sorting. Whether the order will then be stored in yleAPI or in a separate place needs to be decided.

**Comment by mlehtima** [comment]
I renamed one variable as suggested. I think we can now include this and add selecting different sorting orders later.

**Pull request merged**